x-pack/plugins/security_solution_ess/public/navigation/side_navigation.ts

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { map } from 'rxjs';
+import produce from 'immer';
+import { i18n } from '@kbn/i18n';
+import { SecurityPageName, SECURITY_UI_APP_ID } from '@kbn/security-solution-navigation';
+import type { AppDeepLinkId, GroupDefinition, NodeDefinition } from '@kbn/core-chrome-browser';
+import { type Services } from '../common/services';
+
+export const SOLUTION_NAME = i18n.translate('xpack.securitySolutionEss.nav.solutionName', {
+  defaultMessage: 'Security',
+});
+
+export const initSideNavigation = async (services: Services) => {
+  const { securitySolution, navigation } = services;
+
+  navigation.isSolutionNavEnabled$.subscribe((isSolutionNavigationEnabled) => {
+    securitySolution.setIsSolutionNavigationEnabled(isSolutionNavigationEnabled);
+  });
+
+  const { navigationTree$, panelContentProvider } = await securitySolution.getSolutionNavigation();
+
+  const essNavigationTree$ = navigationTree$.pipe(
+    map((navigationTree) =>
+      produce(navigationTree, (draft) => {
+        const footerGroup: GroupDefinition | undefined = draft.footer?.find(
+          ({ type }) => type === 'navGroup'
+        ) as GroupDefinition;
+        const management = footerGroup?.children.find((child) => child.link === 'management');
+        if (management) {
+          management.renderAs = 'panelOpener';
+          management.id = 'stack_management';
+          management.spaceBefore = null;
+          management.children = stackManagementLinks;
+        }
```

Comment:
This injects the static "stack management" links in a hacky way, this code is temporary and its only purpose is to showcase the panel flyout for this section. 

Since the new navigation will be disabled for all customers on 8.14 (it will only be available to internal Elastic users), we can merge this code as is. Once a decision is made on how to show "stack management" links this code will be cleaned.